src/transformers/models/longt5/modeling_longt5.py

```diff
@@ -247,6 +248,8 @@ def forward(self, hidden_states):
         return self.weight * hidden_states
 
 
+ALL_LAYERNORM_LAYERS.append(LongT5LayerNorm)
```

Comment:
This should be after the try/except block below, which can change `LongT5LayerNorm`.